### PR TITLE
Admin page and Admin page manage order

### PR DESCRIPTION
**Pull request by phantuananh1206**

### CODE REVIEW

**Comment by phantuananh1206** [comment]
Admin page and Admin page manage order

**Comment by phantuananh1206** [comment]
image

**Comment by phantuananh1206** [comment]
image

**Comment by phantuananh1206** [comment]
image

**Review by truong295 — APPROVED** [review]

**Comment by phantuananh1206** [comment]
ready

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Admin::OrdersController < Admin::BaseController
+  before_action :load_order, only: %i(update)
+
+  def index
+    @orders = Order.by_created_at.page(params[:page])
+                   .per(Settings.page.per_page)
+  end
+
+  def update; end
```
Comment (vanvtt-0952) [inline app/controllers/admin/orders_controller.rb]: Xóa code này, nó ko thuộc về pull này chú nhé

Comment (phantuananh1206) [reply app/controllers/admin/orders_controller.rb]: Dạ vâng

**Comment by phantuananh1206** [comment]
ready

**Pull request merged**